internal/custom-code-logging/command_custom_code_logging.go

```diff
@@ -0,0 +1,73 @@
+package customcodelogging
+
+import (
+	"github.com/spf13/cobra"
+
+	cclv1 "github.com/confluentinc/ccloud-sdk-go-v2/ccl/v1"
+
+	pcmd "github.com/confluentinc/cli/v4/pkg/cmd"
+	"github.com/confluentinc/cli/v4/pkg/config"
+	"github.com/confluentinc/cli/v4/pkg/featureflags"
+	"github.com/confluentinc/cli/v4/pkg/output"
+)
+
+type customCodeLoggingCommand struct {
+	*pcmd.AuthenticatedCLICommand
+}
+
+type customCodeLoggingShortOut struct {
+	Id          string `human:"ID" serialized:"id"`
+	Cloud       string `human:"Cloud" serialized:"cloud"`
+	Region      string `human:"Region" serialized:"region"`
+	Environment string `human:"Environment" serialized:"environment"`
+}
+
+type customCodeLoggingOut struct {
+	Id          string `human:"ID" serialized:"id"`
+	Cloud       string `human:"Cloud" serialized:"cloud"`
+	Region      string `human:"Region" serialized:"region"`
+	Environment string `human:"Environment" serialized:"environment"`
+	Destination string `human:"Destination" serialized:"destination"`
+	Topic       string `human:"Topic" serialized:"topic"`
+	ClusterId   string `human:"Cluster ID" serialized:"cluster"`
```

Comment:
Usually we just use Cluster to keep it consistent:

`Cluster        string` `human:"Cluster" serialized:"cluster"`